### ISSUE TITLE
specifying libs with modern gcc

### ISSUE THREAD

**Issue by cbueche** [fenugrec/hp3478a_utils#1]
your program works fine here, thank you ! Small note : modern gcc wants libs later on the gcc command :+1: 

```diff
diff --git a/Makefile b/Makefile
index e8442ad..a82b5fc 100644
--- a/Makefile
+++ b/Makefile
@@ -1,5 +1,6 @@
 CC = gcc
-BASICFLAGS = -std=gnu11 -Wall -Wextra -Wpedantic -lm
+BASICFLAGS = -std=gnu11 -Wall -Wextra -Wpedantic
+LDLIBS = -lm
 OPTFLAGS = -g
 EXFLAGS = -DWITH_GPIB=0
 CFLAGS = $(BASICFLAGS) $(OPTFLAGS) $(EXFLAGS)
```

**Comment by fenugrec** [comment]
hm, strange ; I can probably change it without breaking older gcc but do you have more info on when / which gcc version require this ?

**Comment by cbueche** [comment]
Not certain, at least with gcc 9.2.1 on Kubuntu 18.04 it worked fine only when "-lm" was shifted after the `*.c` file. Not sure since when `LDLIBS` is supported but certainly already since a long time. I remember using similar options like `LDFLAGS` in 2003 or so.

**Comment by fenugrec** [comment]
Indeed, I'm not too worried about any reasonably modern gcc > 4.9. In any case it's definitely a linker option and doesn't belong at the compilation step. Fixed in 726220b , thanks !
